plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -94,7 +94,7 @@ read_location (const FuThunderboltFwLocation  *location,
 		g_set_error (error,
 			     FWUPD_ERROR, FWUPD_ERROR_READ,
 			     "Given location is outside of the given FW (%s)",
-			     location->description);
+			     location->description ? location->description : "N/A");
```

Comment:
Maybe it worth adding description to the other `FuThunderboltFwLocation` instead? Because `N/A` isn't much better than `null`.

Reply:
You know this code so much better, I would let you decided if you can fill in more relevant information in the `.description`. I am just not a fan of passing NULL pointers to g_print (I remember this used to crash on Sun).

Reply:
Fair enough.